core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -13,16 +13,23 @@
         height: 100%;
     }
     .p-tabview-panels {
+        // hiding because it is add content type tab there is no need to show this content
+        p-tabpanel:first-child {
+            display: none;
+        }
         height: 100%;
         flex-grow: 1;
         flex-basis: 0;
         overflow: auto;
     }
+
+    .p-tabview-nav-content {
+        border: 1px solid $input-border-color;
+    }
     .p-tabview-nav {
         padding: 0;
-        background: $white;
-        border: 1px solid $input-border-color;
-        border-bottom: none;
+        // background: $white;
+        // border-bottom: none;
```

Comment:
it is necessary leave this comments here?

Reply:
updated